lib/blade/sauce_labs_plugin.rb

```diff
@@ -48,11 +48,11 @@ def config
   end
 
   def username
-    config.username || ENV["SAUCE_USERNAME"]
+    ENV["SAUCE_USERNAME"] || config.username
   end
 
   def access_key
-    config.access_key || ENV["SAUCE_ACCESS_KEY"]
+    ENV["SAUCE_ACCESS_KEY"] || config.access_key
```

Comment:
Flip so environment-provided credentials always override whatever's in `.blade.yml` config